src/frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ashley",
-  "version": "1.1.2",
+  "version": "1.2.0",
```

Comment:
you forgot to change the version in `setup.cfg`

Reply:
my mistake the two pr were linked and setup.cfg is already in 1.2.0 on master
https://github.com/openfun/ashley/commit/479a1270f1827ee7d21d5f1cc71da046ca7f27cd

Reply:
ah ok :ok_hand: